packages/web-config-server/src/apiV1/dashboards.js

```diff
@@ -16,36 +16,41 @@ export default class extends RouteHandler {
     const permissionGroups = await this.req.getUserGroups(entityCode);
     const hierarchyId = await this.fetchHierarchyId();
     // Fetch dashboards
-    const dashboards = await this.models.dashboard.getDashboards(
-      entity,
-      hierarchyId,
-      query.projectCode,
-    );
+    const dashboards = await this.models.dashboard.getDashboards(entity, hierarchyId);
     const sortedDashboards = orderBy(dashboards, ['sort_order', 'name']);
     // Fetch dashboard items
-    const dashboardsWithItems = await Promise.all(
-      Object.values(sortedDashboards).map(async dashboard => {
-        const dashboardItems = await this.models.dashboardItem.fetchItemsInDashboard(
-          dashboard.id,
-          permissionGroups,
-        );
-        const sortedDashboardItems = orderBy(dashboardItems, ['sort_order', 'code']);
-        return {
-          dashboardName: dashboard.name,
-          dashboardId: dashboard.id,
-          dashboardCode: dashboard.code,
-          entityType: entity.type,
-          entityCode,
-          entityName: entity.name,
-          items: Object.values(sortedDashboardItems).map(item => ({
-            code: item.code,
-            legacy: item.legacy,
-            reportCode: item.report_code,
-            ...item.config,
-          })),
-        };
-      }),
-    );
+    const dashboardsWithItems = (
+      await Promise.all(
+        Object.values(sortedDashboards).map(async dashboard => {
```

Comment:
Hmm. Looking back on it this `Object.values` isn't actually _doing_ anything because `sortedDashboards` is an array. This one's definitely my fault 😅 

```suggestion
        sortedDashboards.map(async dashboard => {
```